src/games/strategy/triplea/ui/NotesPanel.java

```diff
@@ -1,20 +1,15 @@
 package games.strategy.triplea.ui;
 
-import java.awt.Component;
 import java.awt.Rectangle;
 
 import javax.swing.BoxLayout;
-import javax.swing.JButton;
 import javax.swing.JEditorPane;
-import javax.swing.JLabel;
 import javax.swing.JPanel;
 import javax.swing.JScrollPane;
-import javax.swing.SwingUtilities;
 
 public class NotesPanel extends JPanel {
   private static final long serialVersionUID = 2746643868463714526L;
   protected final JEditorPane m_gameNotesPane;
-  final JButton m_refresh = new JButton("Refresh Notes");
```

Comment:
Because I think it's not actually working... The only thing it does, is to reinitialize the Panel with the same Variables as before - no change will happen due to [your Commit in the Past](https://github.com/triplea-game/triplea/commit/447e0f4064053afe6cb09b00409c1347459cd6ce)